src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -35,10 +35,10 @@ angular
  * There is an example below of how this should look.
  *
  * ### Notes
- * The `md-autocomplete` uses the the [VirtualRepeat](api/directive/mdVirtualRepeatContainer)
+ * The `md-autocomplete` uses the the <a ng-href="api/directive/mdVirtualRepeatContainer">VirtualRepeat</a>
```

Comment:
I'm not sure, but I've been using it for a while. Not sure if it would work in our case, because of the different versions in the URL. Might be worth a shot though.